src/gmail.js

```diff
@@ -3955,14 +3955,14 @@ var Gmail = function(localJQuery) {
            options.flat  boolean if true will just return an array of all recipients instead of splitting out into to, cc, and bcc
         */
         recipients: function(options) {
-            if( typeof options !== "object" ) options = {};
+            if (typeof options !== "object") options = {};
             const peopleKit = api.check.is_peoplekit_compose(this.$el);
 
             const type_selector = options.type ? "[name=" + options.type + "]" : "";
 
             const found = peopleKit ?
-                this.$el.find("tr.bzf " + type_selector + " div[data-hovercard-id]").map((_, el) => ({
-                    type: el.closest("div[name]").getAttribute("name"),
+                this.$el.find("tr.bzf " + (type_selector || "div[name]") + " div[data-hovercard-id]").map((_, el) => ({
+                    type: options.type || el.closest("div[name]").getAttribute("name"),
```

Comment:
Could another option here be to reverse this statement, using a null coalescing operator?

`````js
type: el.closest("div[name]")?.getAttribute("name") || options.type,
`````

Or even more: Is this change really needed at all? with the previous fix? :)

Reply:
This change allows us to avoid traversing the DOM, therefore providing result faster. If we use specific `"div[name=to]"` parent selector, there is no need to look up for the parent, and then taking its `name` attribute. 🙂

Reply:
Fair enough!